packages/coinstac-ui/config/development.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I'm fine w/ renaming this, but there is already a `local-dev` file

Reply:
I'll get rid of these. I think these are remnants of an earlier age.